imgui-opengl/src/nativeMain/kotlin/com/imgui/impl/ImGuiOpenGL3.kt

```diff
@@ -227,21 +264,12 @@ actual class ImGuiOpenGL3 actual constructor(
 		val lastEnableCullFace = glIsEnabled(GL_CULL_FACE)
 		val lastEnableDepthTest = glIsEnabled(GL_DEPTH_TEST)
 		val lastEnableScissorTest = glIsEnabled(GL_SCISSOR_TEST)
-		val clipOriginLowerLeft = if (useClipOrigin) {
-			val lastClipOrigin = glGetInteger(GL_CLIP_ORIGIN)
-			lastClipOrigin != GL_UPPER_LEFT.toInt()
-		} else {
-			true
-		}
+		val clipOriginLowerLeft = if (!glHasClipOrigin) true else glGetInteger(GL_CLIP_ORIGIN) != GL_UPPER_LEFT.toInt()
```

Comment:
This one is different.

```suggestion
		val clipOriginLowerLeft = !glHasClipOrigin || glGetInteger(GL_CLIP_ORIGIN) != GL_UPPER_LEFT.toInt()
```